app/templates/privacy-and-data.html

```diff
@@ -33,24 +34,24 @@ <h2>What information is collected and how it is used</h2>
           <li>professional or functional anchor</li>
           <li>grade</li>
         </ul>
-        <p>People signing up to take part in the mentoring programme have given explicit consent for their data to be used for the purpose of matching mentors and mentees.</p>
-        <p>To enable the mentor matcher to function, all of this data is uploaded to a server and processed. Once mentors and mentees are matched together, users of this service can download the data. The data is deleted once it has been downloaded.</p>
-        <p>No further consent is required to use this tool, as it is an essential part of the programme.</p>
-        <p>Special category data is collected as part of the programme. This service processes special category data in order to make the programme function. Where special category data is collected and processed, explicit consent has been gained from the data subject.</p>
+        <p>People signing up to take part in the mentoring programme must have given explicit consent for their data to be used for the purpose of matching mentors and mentees. <strong>If this consent has not been obtained, you should not use this service to match mentors and mentees.</strong></p>
+        <p>To enable the mentor matcher to function, all of the data collected as part of the programme is uploaded to a server and processed. Once mentors and mentees are matched together, users of this service can download the data. The data is deleted once it has been downloaded.</p>
```

Comment:
NB this will change again once we've done the notify stuff, as it won't be deleted until the user says they're finished, the emails have been sent, or 15 minutes have elapsed.

But it's good for now!